src/D3D/D3DDebugLogger.cpp

```diff
@@ -46,16 +40,22 @@ D3DDebugLogger::D3DDebugLogger(const Args& arguments) : m_callBackCookie{ 0u } {
 	else {
 		std::ofstream{ "ErrorLog.txt", std::ios_base::app | std::ios_base::out } << std::format("Something went wrong while initializing debugInfoQueue. HRESULT:0x{0:X}", static_cast<unsigned>(hr)).c_str() << std::endl;
 	}
+#else
+	OutputDebugStringA("Failed to initalize D3D12DebugLogger: ID3D12InfoQueue1 interface is not defined. Upgrade to Windows 11 or use DX12 Agility SDK from NuGet.\n");
+	std::ofstream{ "ErrorLog.txt", std::ios_base::app | std::ios_base::out } << "Failed to initalize D3D12DebugLogger: ID3D12InfoQueue1 interface is not defined. Upgrade to Windows 11 or use DX12 Agility SDK from NuGet." << std::endl;

```

Comment:
Duplicate string. Should have put it into a variable/constant and used that instead.